dm/simulator/internal/mcp/errors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mcp
+
+import (
+	"github.com/pingcap/errors"
+)
+
+var (
+	// ErrMCPCapacityFull means the capacity of the modification candidate pool (MCP) is full.
+	ErrMCPCapacityFull = errors.New("the capacity of the modification candidate pool is full")
```

Comment:
I put the errors here because the simulator is a relatively independent project now.  So I put all the error definitions in the project scope.  We can move the error definitions into the `pkg/errors` further if needed.